src/bin/decode.rs

```diff
@@ -120,10 +120,12 @@ fn main_aux() {
 
     progress!(quiet, "Pretty-printing");
     let printer = Shift::try_new().expect("Could not launch Shift");
-    let source = printer
+    let escaped_source = printer
         .to_source(&spec, json)
         .expect("Could not pretty-print");
 
+    let source = binjs::io::escaped_wtf8::to_unicode_escape(escaped_source);
```

Comment:
This is fixing just the binary, but not the library API. Could it be moved into `to_source`?